jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetSamDebugSupport.kt

```diff
@@ -141,11 +144,13 @@ class DotNetSamDebugSupport : SamDebugSupport {
         val scheduler = RdDispatcher(debuggerLifetime)
         val startInfo = createNetCoreStartInfo(state)
 
+        val debugHostAddress = InetAddress.getByName(debugHost)
+
         val protocol = Protocol(
             serializers = Serializers(),
             identity = Identities(IdKind.Client),
             scheduler = scheduler,
-            wire = SocketWire.Client(debuggerLifetime, scheduler, port = frontendPort, optId = "FrontendToDebugWorker"),
+            wire = SocketWire.Client(debuggerLifetime, scheduler, hostAddress = debugHostAddress, port = frontendPort, optId = "FrontendToDebugWorker"),
```

Comment:
@sdubov Do I need to pass host anywhere for the backend port?

Reply:
We start debugger worker in Server mode. It starts listening for any host to connect when we start. Should work as is.